app/src/main/res/xml/root_preferences.xml

```diff
@@ -8,6 +8,11 @@
             android:entryValues="@array/mode"
             android:key="@string/setting_key_theme"
             android:title="@string/setting_theme" />
+
+        <CheckBoxPreference
```

Comment:
Hey, thanks for your contribution! Could we change this to a list preference with the label "time format" and values "absolute time" and "relative time"?

Reply:
Sure, I'll take care of this !

Reply:
Given there are only **two** choices wouldn't it be better to keep a checkbox? Check-box offers single-click solution while selecting from a list would require two clicks.

Reply:
I think changing settings isn't something that is done frequently, thus, I
prefer ui elements that are more descriptive than click efficient.

Presenting a user two options from which they can choose is IMO better than
hiding the default option.

Reply:
Using check-box would be consistent with other places in Android. Not saying it's a convention but at least that's what I noticed. For example `Date and time` settings:
![image](https://user-images.githubusercontent.com/10272121/107697606-59fea480-6cab-11eb-9b47-ad63575a2b3c.png)

Reply:
I agree with both of you on this, on the one hand as @jmattheis said you don't change settings frequently so an extra click for configuration isn't that much in my opinion. On the other hand, the default time displayed in the web app is always relative and there is no way to change it, making it the *default* one, so a checkbox overriding that *default* setup makes sense too.

I think I will do it with a **ListPreference** though, it's definitely more explicit, and user still have the possibility to toggle between the two modes just by clicking on the TextView.